api/allIncidents/allincidentsRouter.js

```diff
@@ -0,0 +1,200 @@
+const express = require('express');
+const router = express.Router();
+const Incidents = require('./incidentsModel');
+const {
+  checkIncidentExists,
+  validateAndSanitizeIncidentQueries,
+} = require('../middleware');
+
+// TODO document shape of objects coming and going
```

Comment:
Looks like you did this and did it well!

Reply:
Yeah it was helpful having the older one to reference!